api/utils/utils.go

```diff
@@ -13,6 +13,11 @@ import (
 	"sigs.k8s.io/yaml"
 )
 
+// NewPointer returns a pointer to a new value instance
+func NewPointer[T any](v T) *T {
+	return &v
+}
+
```

Comment:
```suggestion
// NewUInt64Pointer returns pointer on a new uint64 value instance
func NewUInt64Pointer(i uint64) *uint64 {
	return &i
}
```
This NewPointer is solely used for uint64 as a new "case", while all other uses can be replaced with existing `NewInt32Pointer`, `NewBoolPointer`, but no strong opinion

Reply:
I'd rather migrate others to the new one in the future than keep all this functions